docs/activities/test-resources/index.md

```diff
@@ -0,0 +1,33 @@
+# Resources for Testing
+
+## Test Environments
```

Comment:
You'll want a blank new line under any level of heading.

```
## Test Environments

There are a number of test instances....
```

Reply:
done